lib/local_system.rb

```diff
@@ -52,7 +52,7 @@ def validate_machinery_compatibility
 It still could work very well. If you run into issues or would like to provide us feedback, you are welcome to file an issue at https://github.com/SUSE/machinery/issues/new or write an email to machinery@lists.suse.com.
 Oficially supported operating systems are: '#{supported_oses}'
 
-To disable this message in the machinery configuration use 'machinery config perform-support-check false'
+To disable this message in the machinery configuration use 'machinery config perform-support-check=false'
```

Comment:
Can you replace `machinery` by `#{$0}` so it gets extended. It is not directly related to your pr but if you change the line anyway.